contracts/mainnet/connectors/morpho-aave/interface.sol

```diff
@@ -0,0 +1,61 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+interface IMorphoCore {
+	function supply(
+		address _poolTokenAddress,
+		address _onBehalf,
+		uint256 _amount
+	) external;
+
+	function supply(
+		address _poolToken,
+		address _onBehalf,
+		uint256 _amount,
+		uint256 _maxGasForMatching
+	) external;
+
+	function borrow(address _poolTokenAddress, uint256 _amount) external;
+
+	function borrow(
+		address _poolToken,
+		uint256 _amount,
+		uint256 _maxGasForMatching
+	) external;
+
+	function withdraw(address _poolTokenAddress, uint256 _amount) external;
+
+	function repay(
+		address _poolTokenAddress,
+		address _onBehalf,
+		uint256 _amount
+	) external;
+
+	function claimRewards(
+		address[] calldata _tokenAddresses,
+		bool _tradeForMorphoToken
+	) external;
+}
+
+interface IMorphoAaveLens {
+	function _getCurrentBorrowBalanceInOf(address _poolToken, address _user)
+		external
+		view
+		returns (
+			address underlyingToken,
+			uint256 balanceInP2P,
+			uint256 balanceOnPool,
+			uint256 totalBalance
+		);
+
+	function _getCurrentSupplyBalanceInOf(address _poolToken, address _user)
```

Comment:
For those 2 functions they will be deprecated in a future upgrade of the Lens. I strongly recommend you refer to the this [interface](https://github.com/morpho-dao/morpho-core-v1/blob/main/contracts/aave-v2/interfaces/ILens.sol). 
`getCurrentSupplyBalanceInOf` and `getCurrentBorrowBalanceInOf`

Reply:
Yes, updated the functions in the interface!